packages/template-retail-react-app/package.json

```
@@ -1,5 +1,5 @@
 {
-    "name": "retail-react-app",
+    "name": "scaffold-pwa",
```

```suggestion
    "name": "template-retail-react-app",
```

I assume you did this because the tooling looks here for the name. But we should update the tooling, not this name. :P